binding.gyp

```diff
@@ -22,7 +22,7 @@
         "src/deasync.cc"
       ],
       "include_dirs": [
-        "<!@(node -p \"require('node-addon-api').include\")"
+        "<!@(node -p \"require('node-addon-api').include.replace(/\\\s/g, \\\"\\\\\\\\\\\\\\\ \\\")\")"
```

Comment:
That's a lot of backslashes, I know. It translates to executing this command:

```
node -p "require('node-addon-api').include.replace(/\\s/g, \"\\\\\ \")" 
```

which changes `foo bar baz` to `foo\ bar\ baz`, which does the trick.